docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
+services:
+  instances:
+    build: .
+    restart: unless-stopped
+    ports:
+      - "127.0.0.1:3000:3000"
```

Comment:
Port 3000 is a bad idea considering the fact that invidious also run on port 3000. It's not really desirable if users decide to host both projects on the same machine. 3001 (or something else) would be better.

Reply:
That's just an example port, for example used for local development. For production the user would use a pre-built image from dockerhub and needs to edit the compose file anyways.